website/docs/r/machine_learning_workspace_network_outbound_rule_service_tag.html.markdown

```diff
@@ -0,0 +1,112 @@
+---
+subcategory: "Machine Learning"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_machine_learning_workspace_network_outbound_rule_service_tag"
+description: |-
+  Manages an Azure Machine Learning Workspace Service Tag Network Outbound Rule .
```

Comment:
The resource is called network outbound rule service tag, but here we're saying manages a service tag network outbound rule. Is this a tag or a rule?
```suggestion
  Manages an Azure Machine Learning Workspace Service Tag Network Outbound Rule.
```

Reply:
Service Tag is a type of the network rule. Others is FQNN, Private Endpoint